frontend/simple/js/state.js

```diff
@@ -179,4 +201,11 @@ const actions = {
 const debouncedSave = debounce(dispatch => dispatch('saveSettings'), 500)
 
 store = new Vuex.Store({state, mutations, getters, actions})
+async function loadLastUser () {
+  let user = await db.loadCurrentUser()
+  if (user) {
+    store.dispatch('login', user)
+  }
+}
+loadLastUser()
```

Comment:
If the contents of such a tiny function is ever only ever going to be used once, it makes sense to not make it a function. *"Save The Lines!"* (insert picture of activist holding sign) whenever you can.

I'm also not sure whether this should be done here or in main.js, but you're right that it's probably one of those files.